wqu_app/message.py

```diff
@@ -30,10 +30,17 @@ def get_weather(coords):
 
 
 def greet(ip_address):
-    coords, city = get_geolocation(ip_address)
+    coords, city, region = get_geolocation(ip_address)
     temperature = get_weather(coords)
 
-    return f"Hello, the temperature in {city} (based on your IP address) is {temperature} deg C"
+    if region == None:
+    	return f"Hello, the temperature in {city} (based on your IP address) is {temperature} deg C"
+    elif region == " ":
+    	return f"Hello, the temperature in {city} (based on your IP address) is {temperature} deg C"
+    elif city not in region:
+    	return f"Hello, the temperature in {city}, {region} (based on your IP address) is {temperature} deg C"
+    else:
+    	return f"Hello, the temperature in {city} (based on your IP address) is {temperature} deg C"
```

Comment:
Do you think there's a way to simplify all these checks? You might be able to do so if you combine the city and region data into one string or one object/variable.